var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -174,11 +174,13 @@ class Openmpi(AutotoolsPackage):
     patch('configure.patch', when="@1.10.1")
     patch('fix_multidef_pmi_class.patch', when="@2.0.0:2.0.1")
 
+    fabrics = ('psm', 'psm2', 'pmi', 'verbs', 'mxm', 'libfabric')
```

Comment:
openmpi 3 still says it supports mxm (i.e. the configure script still has the option).  I'm not that familiar with mxm/ucx, so I'm hesitant to remove that option in case it's being used by someone.
Ucx support is there as a variant, though it may make more sense to be an option in the fabrics list. @junghans  - looks like you added the +ucx variant. Have an opinion on that ?

Reply:
@baberlevi - ypu are right, it supports ucx and mxm. Probably we should have both options htere. ucx is a new layer that gradually replacing mxm.